CompanyManagementBusinessLayer/DataConverter.cs

```diff
@@ -55,7 +55,7 @@ public BODepartmentMaster BODepartmentDao(DepartmentMaster department)
             };
             return boDepartmentMaster;
         }
-        public BOEmployee BOEmployeeDao(Employee employee)
+        public BOEmployee EmployeeConverter(Employee employee)
```

Comment:
BOEmployee
{
int EmployeeID {get;set;}
string EmployeeName {get;set;}
Department Department {get;set;}

}

either
in businesLayer
BOEmployee emp= DataConvertor.Convert(DO Employee)
emp.Department = Datalayer.GetDepartment(int departmentID)


Or dataconvertor itself
emp.Department = datalayer.GetDepatment(int departmentID)